src/Common/nunit/ConsoleOptions.cs

```diff
@@ -382,8 +382,10 @@ private void ConfigureOptions()
             this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
                 v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
 
+#if !NETCF
```

Comment:
Sorry, I do get confused sometime, since my console runner does build for NETCF.  Just like my NETCF build of the framework has the FrameworkController in it. 

 

However, ConsoleOptions currently only has NETCF conditionals in it for CF platform related issues, like TryParse methods.

 

I thought that at one point, that was discussed as a desirable thing.  But that was a while ago.

 

From: CharliePoole [mailto:notifications@github.com] 
Sent: Monday, April 27, 2015 6:14 AM
To: nunit/nunit
Cc: Neil Colvin
Subject: Re: [nunit] Issue #611: Remove unused --teamcity option from CF build of NUnitLite (#619)

 

In src/Common/nunit/ConsoleOptions.cs <https://github.com/nunit/nunit/pull/619#discussion_r29114815> :

> @@ -382,8 +382,10 @@ private void ConfigureOptions()
>              this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
>                  v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
>  
> +#if !NETCF

Actually, #if !NETCF is exactly how we have done all these conditions. This is a file that is shared between the console runner and NUnitLite. Since the console runner is only built for desktop .NET, NETCF always implies NUNITLITE. We could test for both, but that's not how we have done it elsewhere. Of course, if we decided to build a full console runner for CF, we would end up changing many conditions.

—
Reply to this email directly or view it on GitHub <https://github.com/nunit/nunit/pull/619/files#r29114815> .  <https://github.com/notifications/beacon/AFh64aOV-gwFO_lcrTKdfrbkkArkixg9ks5oDVq3gaJpZM4EJMBn.gif>